File: recognitiontests/java/benchmark/presentation/exception/LinkedInException.java
package presentation.exception;

import domain.linkedin.LinkedInDTO;

public class LinkedInException extends Exception {

	public LinkedInException(LinkedInDTO transferAccount) {
		super("name " + transferAccount.getName() + " password: " + transferAccount.getPassword());
	}

	public LinkedInException(String message) {
		super(message);
	}
}

File: testprojects/c#/presentation/exception/LinkedInException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.linkedin;

namespace CSharpBenchmark.presentation.exception
{

    public class LinkedInException : Exception
    {

        public LinkedInException(LinkedInDTO transferAccount)
            : base("name " + transferAccount.getName() + " password: " + transferAccount.getPassword())
        {
        }

        public LinkedInException(String message)
            : base(message)
        {
        }
    }
}